dll/win32/browseui/internettoolbar.cpp

```diff
@@ -1086,12 +1086,8 @@ HRESULT CInternetToolbar::ToggleBandVisibility(int BandID)
     REBARBANDINFOW bandInfo = {sizeof(REBARBANDINFOW), RBBIM_STYLE};
     SendMessage(fMainReBar, RB_GETBANDINFOW, index, (LPARAM)&bandInfo);
 
-    if (bandInfo.fStyle & RBBS_HIDDEN)
-        bandInfo.fStyle &= ~RBBS_HIDDEN;
-    else
-        bandInfo.fStyle |= RBBS_HIDDEN;
-
-    SendMessage(fMainReBar, RB_SETBANDINFOW, index, (LPARAM)&bandInfo);
```

Comment:
The fact that the original code didn't work could also indicate a bug in the rebar code implementation in Wine's comctl32...

Reply:
Well, at least with this change the code works, it's not a bug from this problem, and with this method the code seems more cleaner

Reply:
i could check if there is a bug on rebar anyways

Reply:
and im not sure that this is a bug in rebar, if it is the change needed is to call showwindow inside RB_SETBANDINFOW callback, that the unique thing RB_SHOWBAND does